src/main/kotlin/org/jetbrains/kotlinx/jupyter/repl.kt

```diff
@@ -420,10 +424,11 @@ class ReplForJupyterImpl(
             notebook.updateVariablesState(internalEvaluator)
             // printVars()
             // printUsagesInfo(jupyterId, cellVariables[jupyterId - 1])
+            val entry = notebook.variablesState.entries.lastOrNull()
+            val serializedVarsState =  variablesSerializer.serializeVariableState(jupyterId - 1, entry?.key, entry?.value)
```

Comment:
Where is this variable used?

Reply:
Nothing, just for test)

Reply:
Let's then remove it, tests should go to tests sources :)